site/content/data-platform/_index.md

```diff
@@ -21,6 +21,52 @@ AI solutions for GraphRAG, graph machine learning, data explorations, and more.
 run it on-premises or in the cloud yourself on top of Kubernetes to access all
 of the platform features with enterprise-grade automation and reliability.
 
+## What Makes Up the Arango Data Platform
+
+The Arango Data Platform is built on a layered architecture that combines powerful
+components into a unified solution:
+
+- **ArangoDB Enterprise Edition**: The multi-model database foundation supporting
+  graphs, documents, key-value, vector search, and full-text search capabilities.
+
+- **Graph Visualizer**: A sophisticated web-based interface for graph exploration,
+  smart search, and visual layouts.
+
+- **Arango Platform Suite**: Enterprise-grade features including high availability
+  and monitoring, the Graph Analytics Engine, comprehensive APIs and connectors,
```

Comment:
Graph analytics → Part of AI Suite in the October release isn't it?

Reply:
Fixed.